felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -228,7 +236,7 @@ type bpfEndpointManager struct {
 	rpfStrictModeEnabled string
 
 	// Service routes
-	ctlbWorkaroundEnabled bool
+	ctlbWorkaroundEnabled ctlbWorkaroundState
```

Comment:
```suggestion
	ctlbWorkaroundMode ctlbWorkaroundMode
```

Reply:
"Enabled" has connotations of being a bool so I think it's worth changing to say Mode.